plugins/filesystem/local/adapter/adapter.php

```diff
@@ -81,6 +84,98 @@ public function getFile($path = '/')
 	}
 
 	/**
+	 * Returns the folder or file information for the given path. The returned object
```

Comment:
Did you modify something in that code? There is for me no reason why it should be changed. For me it looks identical.